.github/workflows/ci.yml

```diff
@@ -0,0 +1,63 @@
+name: CI
+
+on:
+  push:
+    branches-ignore:
+      - 'dependabot/**'
+  pull_request:
+
+env:
+  CI: true
+  FORCE_COLOR: 2
+
+jobs:
+  run:
+    name: Node ${{ matrix.node }}
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        node:
+          - 10
+          - 12
+          - 14
+          - 15
+
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v2
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: '${{ matrix.node }}'
+
+      - name: Set up npm cache
+        uses: actions/cache@v2
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-node-v${{ matrix.node }}-${{ hashFiles('package.json') }}-${{ hashFiles('package-lock.json') }}
+          restore-keys: |
+            ${{ runner.OS }}-node-v${{ matrix.node }}-${{ hashFiles('package.json') }}-${{ hashFiles('package-lock.json') }}
+            ${{ runner.OS }}-node-v${{ matrix.node }}-
+
+      - name: Install npm dependencies
+        run: npm ci
+
+      - name: Run Jest
+        run: npm run test:jest
+        if: matrix.node != 14
+
+      - name: Run Jest with coverage
+        run: npm run test:jest:cov
+        if: matrix.node == 14
```

Comment:
Would it make sense to move this to an env var, so that there is only a single reference to the Node version used for coverage?

Reply:
Can do, how do you want to name the env var? Also, do you want the env var to be in the global scope or under the job?

Reply:
The pattern in the site action is great, maybe just copy that?

Reply:
OK, it's just that it might not be self-explanatory using `NODE`, but I can add a comment.

Also, we could just run jest with `--coverage` regardless of the Node version, and just run coveralls only on Node.js 14. That would be less lines.

Reply:
Saw you changed the name to NODE_COV, which should be easy enough to distinguish.

> we could just run jest with --coverage regardless of the Node version

I like that idea as well. But the current slight duplication isn't too bad either 🙂

Reply:
Your call. There isn't so much of a difference, just 3 seconds. I'll leave it as is and you can change it later if needed.